mlrun/runtimes/daskjob.py

```diff
@@ -588,7 +595,10 @@ def enrich_dask_cluster(function, secrets, client_version):
     spec.remote = True
 
     image = (
-        function.full_image_path(client_version=client_version) or "daskdev/dask:latest"
+        function.full_image_path(
+            client_version=client_version, client_python_version=client_python_version
+        )
+        or "daskdev/dask:latest"
```

Comment:
👀 question from the crowd. in which cases we fallback to `daskdev/dask:latest`?

Reply:
This is in case where no image was defined in the function's spec image, which I currently I don't see how we can get into that case, I guess this might be a left over from the time the client was responsible on submitting jobs and not the server. Will add TODO

Reply:
great, comment that? :)